web/themes/custom/pineapple/sass/base/_buttons.scss

```diff
@@ -0,0 +1,29 @@
+.js-form-submit.form-submit {
```

Comment:
As discussed use the button class and not drupal generated classes i.e `.js-form-submit`

Reply:
done